docs/plugins/analysis-kuromoji.asciidoc

```diff
@@ -172,6 +172,8 @@ PUT kuromoji_sample
   }
 }
 
+GET _cluster/health?wait_for_status=yellow
```

Comment:
can we do this wait as part of the general rest setup? Seems not nice to require we add waits for basically all docs?